web/src/components/PoisDesktop.tsx

```diff
@@ -26,7 +26,6 @@ const PanelContainer = styled.article`
   display: flex;
   flex-direction: column;
   width: ${dimensions.poiDesktopPanelWidth}px;
-  min-width: ${dimensions.poiDesktopPanelWidth}px;
```

Comment:
Was this removed on purpose?

Reply:
Yes, we already have the width for exactly the size right above it so it's unnecessary.

Reply:
Sometimes I still had problems with this as a `width` alone was not enough and the width was less than what was set. I think this was one of the cases.